cumulus/parachains/runtimes/assets/asset-hub-rococo/src/weights/xcm/mod.rs

```diff
@@ -140,14 +141,15 @@ impl<Call> XcmWeightInfo<Call> for AssetHubRococoXcmWeight<Call> {
 		_dest: &Location,
 		remote_fees: &Option<AssetTransferFilter>,
 		_preserve_origin: &bool,
-		assets: &Vec<AssetTransferFilter>,
+		assets: &BoundedVec<AssetTransferFilter, MaxAssetTransferFilters>,
 		_xcm: &Xcm<()>,
 	) -> Weight {
+		let base_weight = XcmFungibleWeight::<Runtime>::initiate_transfer();
```

Comment:
@franciscoaguirre you only added `base_weight` for AHs, all the other chains don't have it